File: JAVA PROGRAMMING/Test59.java
// 59. PRINT THE DIGITS OF A NUMBER (INPUT: 153 OUTPUT: 3 5 1)
import java.util.Scanner;
public class Test {
    public static void main(String args[]) {
        Scanner in = new Scanner(System.in);
        System.out.println("Enter Input Value: ");
        int num = in.nextInt();

        while(num != 0) {
            int rem = num % 10;
            System.out.print(rem+" ");
            num /= 10;
        }
        in.close();
    }
}

File: TYPESCRIPT PROGRAMMING/Test59.ts
// 59. PRINT THE DIGITS OF A NUMBER (INPUT: 153 OUTPUT: 3 5 1)
const digitOfNumber = (input: number): void => {
    while(input !== 0) {
        let rem: number = input % 10;
        process.stdout.write(`${rem} `);
        input = Math.floor(input / 10);
    }
    console.log();
}
digitOfNumber(153);
digitOfNumber(1234);